Apps/W1/HybridGP/app/src/Migration/Accounts/GPAccountMigrator.codeunit.al

```diff
@@ -134,25 +202,25 @@ codeunit 4017 "GP Account Migrator"
 
     local procedure CreateDimSet(GPGLTransactions: Record "GP GLTransactions"): Integer
     var
-        TempDimSetEntry: Record "Dimension Set Entry" temporary;
-        DimVal: Record "Dimension Value";
+        TempDimensionSetEntry: Record "Dimension Set Entry" temporary;
+        DimensionValue: Record "Dimension Value";
         GPSegments: Record "GP Segments";
         HelperFunctions: Codeunit "Helper Functions";
-        DimMgt: Codeunit DimensionManagement;
+        DimensionManagement: Codeunit DimensionManagement;
         NewDimSetID: Integer;
     begin
         if GPSegments.FindSet() then
             repeat
-                DimVal.Get(HelperFunctions.CheckDimensionName(GPSegments.Id), GetSegmentValue(GPGLTransactions, GPSegments.SegmentNumber));      //'0000'); GPGLTransactions ACTNUMBR_1 - 9
-                TempDimSetEntry.Init();
-                TempDimSetEntry.Validate("Dimension Code", DimVal."Dimension Code");
-                TempDimSetEntry.Validate("Dimension Value Code", DimVal.Code);
-                TempDimSetEntry.Validate("Dimension Value ID", DimVal."Dimension Value ID");
-                TempDimSetEntry.Insert(true);
+                DimensionValue.Get(HelperFunctions.CheckDimensionName(GPSegments.Id), GetSegmentValue(GPGLTransactions, GPSegments.SegmentNumber));      //'0000'); GPGLTransactions ACTNUMBR_1 - 9
+                TempDimensionSetEntry.Init();
```

Comment:
I believe CLEAR is better here since we are in the loop. Clear will clear entire records, init only cleans no primary key fields

Reply:
I will note this and make changes in upcoming PRs so we can fully test this.